frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -240,22 +240,24 @@ import { Problem, ContestResult } from '../../linkable_resource';
   },
 })
 export default class UserProfile extends Vue {
-  @Prop() data!: types.ExtraProfileDetails;
+  @Prop() data!: types.ExtraProfileDetails | null;
   @Prop() profile!: types.UserProfileInfo;
   @Prop() profileBadges!: Set<string>;
   @Prop() visitorBadges!: Set<string>;
-  contests = this.data.contests
-    ? Object.values(this.data.contests)
-        .map((contest) => {
-          const now = new Date();
-          if (contest.place === null || now <= contest.data.finish_time) {
-            return null;
-          }
-          return new ContestResult(contest);
-        })
-        .filter((contest) => !!contest)
+  contests = this.data
+    ? this.data.contests
+      ? Object.values(this.data.contests)
+          .map((contest) => {
+            const now = new Date();
+            if (contest.place === null || now <= contest.data.finish_time) {
+              return null;
+            }
+            return new ContestResult(contest);
+          })
+          .filter((contest) => !!contest)
+      : []
     : [];
-  charts = this.data.stats;
+  charts = this.data ? this.data.stats : [];
```

Comment:
```suggestion
  charts: types.UserProfileStats[] = this?.data?.stats ?? [];
```

acá igual. hay que evitar los ternarios cuando se pueda.